app/src/main/java/com/cpr3663/cpr_scouting_app/data/Events.java

```diff
@@ -31,9 +31,9 @@ public class Events {
 
     // Constructor
     public Events() {
-        event_list = new ArrayList<EventRow>();
-        auto_events = new ArrayList<String>();
-        teleop_events = new ArrayList<String>();
+        event_list = new ArrayList<>();
+        auto_events = new ArrayList<>();
+        teleop_events = new ArrayList<>();
```

Comment:
Why? Is it just irrelevant to tell it what it an ArrayList of and if that is the reason why not leave it in to make sure its the right type?

Reply:
Android Studio "complains" that it's unnecessary.  It knows the type, so when you call "new" you don't need to supply it.  I think this is a common Java practice.

Reply:
Oh, ya, I kind of forgot you already told it the type when you defined it.